opencti-platform/opencti-graphql/src/database/redis.ts

```diff
@@ -70,11 +70,22 @@ const redisOptions = (): RedisOptions => ({
   showFriendlyErrorStack: DEV_MODE,
 });
 
-const generateNatMap = (mappings: [{ nat: string; host: string; port: number; }]): Record<string, { host: string; port: number; }> => {
+// From "HOST:PORT" to { host, port }
+export const generateClusterNodes = (nodes: string[]): { host: string; port: number; }[] => {
```

Comment:
I'm wondering if using the delimiter `-` may wrongly parse the host like `a-b-c.redis.com`. Do you think using ` ` (space) or some non-qualified-url character will work also?
I think to avoid the below chars are recommended.

> In general URIs as defined by [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986/) (see [Section 2: Characters](https://www.rfc-editor.org/rfc/rfc3986#section-2)) may contain any of the following 84 characters:
`ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~:/?#[]@!$&'()*+,;=`

Ref. https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid

Reply:
Sure, good remark. we can use a whitespace or other seperator for the nap part.

"HOST:PORT HOST:PORT"

or

"HOST:PORT>HOST:PORT"

Reply:
Changed to "HOST:PORT>HOST:PORT"